Cargo.toml

```diff
@@ -8,6 +8,12 @@ homepage = "https://vector.dev"
 license-file = "LICENSE"
 readme = "README.md"
 publish = false
+default-run = "vector"
+
+[[bin]]
+name = "graphql-schema"
```

Comment:
Introduced this for running schema generation via Cargo:

```bash
cargo run --bin graphql-schema
```

I looked for an existing pattern prior to doing this, to see if there are other entrypoints / helper bins running elsewhere, which didn't yield anything obvious. It doesn't really fit a workspace candidate for `lib` -- it's not really a library, nor does it provide an obvious way to call back into parent deps.

If there is a preferred approach to running helpers that aren't part of the main vector bin (which might include where to keep the source), would appreciate the pointer.

Reply:
Do you intend for this to be a build time thing? It could be part of a build script and run on changed with https://doc.rust-lang.org/cargo/reference/build-scripts.html ?